src/providers/dataProvider/index.ts

```diff
@@ -125,6 +126,25 @@ export const getDataProvider = async (): Promise<DataProvider<string>> => {
     },
     {
       resource: constants.R_BATCHES,
+      beforeUpdate: async (record: UpdateParams<Batch>) => {
+        if (
+          record.previousData.maximumProtectiveMarking !==
+          record.data.maximumProtectiveMarking
+        ) {
+          await audit(
+            AuditType.EDIT_BATCH,
+            `Batch updated (${String(record.data.id)})`,
+            record.previousData.maximumProtectiveMarking !==
+              record.data.maximumProtectiveMarking
+          )
```

Comment:
hmm, the above `if` has already been tested at aline 131.  It can just be `true` here, right?

Reply:
Change implemented in this commit: https://github.com/DeepBlueCLtd/RCO/pull/116/commits/99e5dc4479b89bf9761bb5e90dec2cd7ccdb5c2e